### PR TITLE
*: fix CI.

### PR DESCRIPTION
**Pull request by disksing**
Our internal CI fails with error:
```
+ make test
rm: cannot remove '_vendor/src/github.com/pingcap/pd': Is a directory
```
cc @choleraehyq @nolouch @tiancaiamao

### CODE REVIEW

**Comment by disksing** [comment]
cc @choleraehyq @nolouch @tiancaiamao

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PD_PKG := github.com/pingcap/pd
-VENDOR := $(shell rm -f _vendor/src/$(PD_PKG) &&\
+VENDOR := $(shell rm -rf _vendor/src/$(PD_PKG) &&\
```
Comment (siddontang) [inline Makefile]: is it better to set GOPATH with vendor here directly, so we don't need to link the vendor path anymore?

Comment (disksing) [inline Makefile]: Here is not linking the vendor path but link the pd root path to _vendor. Please refer #679 for more details.

**Comment by nolouch** [comment]
LGTM

**Pull request merged**